UnitsNet.Serialization.JsonNet/UnitsNetJsonConverter.cs

```diff
@@ -41,9 +42,36 @@ public class UnitsNetJsonConverter : JsonConverter
 
             object obj = TryDeserializeIComparable(reader, serializer);
             // A null System.Nullable value or a comparable type was deserialized so return this
-            if (!(obj is ValueUnit vu))
+            if (!(obj is ValueUnit) && !(obj is Array))
                 return obj;
 
+            if (obj is Array)
+            {
+                object[] values = (object[]) obj;
+
+                List<object> results = new List<object>();
+                
+                foreach (var value in values)
```

Comment:
This foreach can be changed to LINQ.

Reply:
True, i copied it from my initial attempt and it was laid out like this for easier debugging while developing.